spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -480,6 +484,8 @@
 "https://raw.github.com/RDFLib/rdflib/master/LICENSE": BSD-3-Clause
 "https://www.eclipse.org/legal/epl-v10.html": EPL-1.0
 "https://www.eclipse.org/legal/epl-v20.html": EPL-2.0
+"https://www.scala-lang.org/license.html": Apache-2.0 AND BSD-3-Clause AND MIT
```

Comment:
Did you check if the license (n)ever changed in the past?

Reply:
Yes, it one of the things I checked, Scala's main license is Apache-2.0 but they have a couple of core components that are BSD-3-Clause AND MIT - within issues I could not find anything that they were planning to rewrite these BSD-3-Clause AND MIT components